app/api_topologies.go

```diff
@@ -255,7 +303,30 @@ type topologyStats struct {
 	FilteredNodes      int `json:"filtered_nodes"`
 }
 
-func (r *registry) add(ts ...APITopologyDesc) {
+// AddContainerFilters adds to the topology Registry's containerFilters
+func AddContainerFilters(newOptions ...APITopologyOption) {
+	topologyRegistry.AddContainerFilters(newOptions...)
+}
+
+// AddContainerFilters adds container filters to this registry
+func (r *Registry) AddContainerFilters(newOptions ...APITopologyOption) {
+	r.Lock()
+	defer r.Unlock()
+
+	for _, key := range []string{containersTopologyDescID, containersByHostnameTopologyDescID, containersByImageTopologyDescID} {
+		var tmp = r.items[key]
+		//for _, optionGroup := range tmp.Options {
+		for i := 0; i < len(tmp.Options); i++ {
+			if tmp.Options[i].ID == "system" {
```

Comment:
If you look at containerFilters, "system" is just the name of the API Topology Option group that contains the label filters. The naming confusion probably is due to containerFilters being named that way before I started adding code